### PR TITLE
App: add progress save toast

### PR DESCRIPTION
**Pull request by 0xGabi**
https://user-images.githubusercontent.com/9082013/128612928-922bd173-f91c-459a-9b55-6bfa480287eb.mov

### CODE REVIEW

**Comment by fioreb** [comment]
The functionality is perfect!
But can we make it look link this: https://www.figma.com/file/ySHlXMwz5elSXKI3feisJU/Gardens-wizard-creation-UI?node-id=543%3A13865 ? only with padding and not that long. Thanks 🙏

File: src/components/Home.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
+import { useToast } from '@1hive/1hive-ui'
```
Comment (fabriziovigevani) [inline src/components/Home.js:1]: Let's place this import after `styled-components` import.

We usually have imports from our own libraries after general library imports.

File: src/components/Home.js [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
 import React, { useCallback, useState } from 'react'
 import styled from 'styled-components'
+
+import { useNodeHeight } from '@hooks/useNodeHeight'
```
Comment (fabriziovigevani) [inline src/components/Home.js:5]: What was the rationale for moving this up? 

We usually have local hooks after components.

**Review by fabriziovigevani — COMMENTED** [review]
🚀 🚀 🙌

**Review by fabriziovigevani — APPROVED** [review]
💪

**Pull request merged**